composer.json

```diff
@@ -46,6 +46,7 @@
         "monolog/monolog": "~1.4,>=1.4.1"
     },
     "suggest": {
+        "silex/web-profiler": "~1.0",
```

Comment:
-1 for this. Displaying a suggestion to anyone using Silex for this is a bad idea IMO. Suggestions are not the place to list additional providers. https://github.com/silexphp/Silex/wiki/Third-Party-ServiceProviders is this place

Suggestions are used here for packages which allow you to use more features from the silex/silex package itself (optional dependencies of the repo).